function.py

```diff
@@ -0,0 +1,89 @@
+from init import db
+from models import Game, User
+import datetime
+from api_connector import ApiConnector
+
+"""
+    Functions that can be carried out by the User.
+"""
+
+
+def create_user(username, password):
+    new_user = User(username=username)
+    new_user.hash_password(password)
+    db.session.add(new_user)
+    db.session.commit()
+
+
+def create_new_game(username):
+    obj = ApiConnector()
+    obj.get_entry()
+    secret_word = obj.get_word()
+    hint = obj.get_definition()
```

Comment:
This is an example of very bad class usage - you require calling so many methods just to get a word and its definition. You can vastly simplify this:
- Remove the ```get_word()``` and ```get_definition()``` methods
- Simply enhance your ```get_entry()``` method to do what it is supposed to do - GET an entry. On return, it should give you an object containing to values: ```Word``` and ```Definition```. 

Bonus: Even better if you can define an actual class and store the information inside it inside of having just an ad-hoc object (using mostly the dynamic aspect of Python when it comes to passing values makes your code very difficult to maintain!)